src/AzureAuth/CommandMain.cs

```diff
@@ -25,12 +25,12 @@ public class CommandMain
     {
         private const string ResourceOption = "--resource";
         private const string ClientOption = "--client";
-        private const string TenantOption = "-t|--tenant";
+        private const string TenantOption = "--tenant";
         private const string ScopeOption = "--scope";
-        private const string ClearOption = "-c|--clear";
-        private const string DomainOption = "-d|--domain";
-        private const string AuthModeOption = "-m|--auth-mode";

```

Comment:
For a future change, I think we might want to have this be just `--mode`. `auth` is redundant.

Reply:
Ah, yeah, that's a great suggestion for a follow-up PR.